xfields/ibs/_equilibrium_emittance.py

```diff
@@ -159,17 +163,28 @@ def _ibs_rates_and_emittance_derivatives(
     )
 
 
+# TODO: find a better, more explicit name for this
 def compute_emittance_evolution(
     twiss: xt.TwissTable,
-    bunch_intensity: float,
-    initial_emittances: tuple = None,
+    formalism: Literal["Nagaitsev", "Bjorken-Mtingwa", "B&M"],
+    total_beam_intensity: int,
+    # TODO: want to force providing gemitt_x, gemitt_y & gemitt_zeta instead of this initial_emittances (can allow nemitt_x and add checks / conversions)
+    initial_emittances: tuple[float, float, float] = None,
+    # gemitt_x: float = None,
+    # nemitt_x: float = None,
+    # gemitt_y: float = None,
+    # nemitt_y: float = None,
+    # gemitt_zeta: float = None,
+    # nemitt_zeta: float = None,
     emittance_coupling_factor: float = 0,
-    emittance_constraint: Literal["Coupling", "Excitation"] = "Coupling",
-    input_sigma_zeta: float = None,
-    input_sigma_delta: float = None,
+    emittance_constraint: Literal["coupling", "excitation"] = "coupling",
+    # TODO: move these two (still optional) below gemitt_x, gemitt_y, gemitt_zeta when modify that
+    overwrite_sigma_zeta: float = None,
+    overwrite_sigma_delta: float = None,
```

Comment:
Slight rename. With this TODO I meant to move the parameters up in the function definition.